src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -2077,7 +2077,7 @@ public class LibraryServiceClient implements AutoCloseable {
    * @param label REQUIRED: The label to add.
    * @throws com.google.api.gax.grpc.ApiException if the remote call fails
    */
-  public final AddLabelResponse addLabel(String resource, String label) {
+   final AddLabelResponse addLabel(String resource, String label) {
```

Comment:
The extra blank space here is OK. It will be stripped by the code formatter when we generate clients "for real"